app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less

```diff
@@ -0,0 +1,233 @@
+h1 {
+    .lib-visually-hidden();
+}
+.section.section__slider {
+    padding-bottom: 30px;
+}
+
+.section {
+    padding-bottom: 30px;
+    h2,
+    h3 {
+        font-size: 24px;
+        font-weight: 600px;
+        letter-spacing: 1px;
+        text-transform: uppercase;
+        color: @color-maksym__main-black;
+    }
+
+    p {
+        font-size: 14px;
+        line-height: 22px;
+        font-weight: 300px;
+        letter-spacing: 1px;
+        color: @color-maksym__main-gray;
+    }
+    button {
+        font-size: 14px;
+        line-height: 22px;
+        font-weight: 300px;
+        letter-spacing: 1px;
+        color: @color-maksym__main-black;
+        &:hover,
+        &:focus {
+            color: inherit;
+        }
+    }
+}
+
+/* SECTION__SLIDER */
+
+.section__slider {
+    .slider {
+        &__title {
+            .lib-visually-hidden();
+        }
+        .pagebuilder-slide-wrapper {
+            display: flex;
+            align-items: center;
```

Comment:
Please use flex mixins, check it here:
`/lib/web/css/source/lib/_utilities.less : line 412`

Reply:
there is a lib for align-items:
`.lib-vendor-box-align()`